app/pages/sections/order-items/associated-services/dashboard/controller.js

```diff
@@ -1,3 +1,16 @@
+import { getData } from 'buying-catalogue-library';
+import { getEndpoint } from '../../../../../endpoints';
+import { logger } from '../../../../../logger';
 import { getContext } from './contextCreator';
 
-export const getAssociatedServicesPageContext = async ({ orderId }) => getContext({ orderId });
+export const getAssociatedServicesPageContext = async ({ orderId, accessToken }) => {
+  const getOrderDescriptionDataEndpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'getDescription', options: { orderId } });
+  const orderDescriptionData = await getData({
+    endpoint: getOrderDescriptionDataEndpoint, accessToken, logger,
+  });

```

Comment:
let's move this guy to the helpers please. And they'll be a few places we can quickly change to do the same

Reply:
Yeah good shout will get this done 👍